src/modules/tests/vehicle-checks/vehicle-checks.reducer.ts

```diff
@@ -17,6 +17,16 @@ export function vehicleChecksReducer(
         tellMeQuestionCode: action.tellMeQuestion.tellMeQuestionCode,
         tellMeQuestionDescription: action.tellMeQuestion.tellMeQuestionDescription,
       };
+    case vehicleChecksActions.TELL_ME_QUESTION_CORRECT:
+      return {
+        ...state,
+        tellMeQuestionOutcome: 'P',
```

Comment:
Could we turn the P, DF, S and D into a enum (may make it easier later on if the code changes, as we are going to be using those values in multiple places)

Reply:
It's already encoded in the `QuestionOutcome` type from the test schema, but you're very correct, it could be useful to have an enum around it aswell. Going to merge this one as-is for now, there's a couple of small tweaks left to finish off this page, we can discuss further and I may try and incorporate this improvement into the next PR.